src/store.ts

```diff
@@ -0,0 +1,30 @@
+import type Book from "./types";
+
+export default class Store {
+
+  static getBooks() {
+    const books: string | null = localStorage.getItem('books')
+    const bookArray = books ? JSON.parse(books) : [];
+    return bookArray;
+  }
+
+  static setItem(books: Book):void {
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static addBook(book: Book):void {
+    const books = Store.getBooks()
+    books.push(book);
+    Store.setItem(books);
+  }
+
+  static removeBook(id: string):void {
+    const books = Store.getBooks();
+    books.forEach((book: Book, index: number) => {
+      if (book.id === id) {
+        books.splice(index, 1)
+      }
+    })
+    Store.setItem(books)
+  }
```

Comment:
why you need book id as object? you can do it only with array, every index can be id for you